### PR TITLE
Sscs 10984 creating new body content for correction

### PR DESCRIPTION
**Pull request by amoslv1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10984

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by StevensonSi — APPROVED** [review]

**Review by jarekPierchala — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PostHearing.java [hunk 1]
```diff
@@ -105,4 +96,13 @@ public RequestFormat getRequestFormat() {
         }
         return null;
     }
+
+    @JsonIgnore
+    public boolean isRefused() {
```
Comment (DGordonV1) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PostHearing.java:101]: Is the REFUSE value always "refuse"? If so, could we turn this into a one liner and just use equals("refuse") on ccdDefinition?

Fine if not, but could reduce some lines.

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/DocumentGeneration.java [hunk 2]
```diff
@@ -19,6 +19,8 @@ public class DocumentGeneration {
     private YesNo generateNotice;
     @JsonProperty("bodyContent")
     private String bodyContent;
+    @JsonProperty("bodyContentCorrection")
```
Comment (DGordonV1) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/DocumentGeneration.java:22]: Is it worth prefixing bodyContent with correction here (if there's any further correction fields to be added here, if not ignore me).

**Review by DGordonV1 — CHANGES_REQUESTED** [review]

**Review by DGordonV1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/DocumentGeneration.java (hunk 2 above)
Comment (amoslv1) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/DocumentGeneration.java:22]: Will consider renaming on a future branch if necessary. We have agreed that this syntax is fine here for now.

**Review by StevensonSi — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/ScannedDocumentType.java [hunk 3]
```diff
@@ -27,4 +28,13 @@ public enum ScannedDocumentType {
         this.label = label;
     }
 
+    public static ScannedDocumentType fromValue(String text) {
+        for (ScannedDocumentType scannedDocumentType : ScannedDocumentType.values()) {
+            if (scannedDocumentType.getValue() != null && scannedDocumentType.getValue().equalsIgnoreCase(text)) {
+                return scannedDocumentType;
+            }
+        }
+        return null;
```
[the remark this answers is not in the document]
Comment (amoslv1) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/ScannedDocumentType.java]: Done

**Review by jarekPierchala — APPROVED** [review]

**Pull request merged**